cloudinit/config/cc_apt_configure.py

```diff
@@ -562,16 +560,18 @@ def is_deb822_format(apt_src_content: str) -> bool:
     Return True if content looks like it is deb822 formatted APT source.
     """
     # TODO(At jammy EOL: use aptsources.sourceslist.Deb822SourceEntry.invalid)
-    try:
-        apt_src = safeyaml.load(apt_src_content)
```

Comment:
This is more refactoring within the series that we don't need in the commit history.

Reply:
+1 on these points here @holman. I had actually squash merged this PR during merge instead of a rebase merge because of the way I had dealt  with review comments and refactoring code after the fact. When I handle some of the review comments, I didn't squash the content into the correct commit in some cases and rather that presenting 10 separate commits for this effort, I thought it was going to be more appropriate to squash merge and represent the set of changes in the single commit message.